storybook/storybook-react/src/LinkList/LinkList.docs.mdx

```diff
@@ -0,0 +1,35 @@
+import { Canvas, Markdown, Meta, Primary } from "@storybook/blocks";
+import * as LinkListStories from "./LinkList.stories.tsx";
+import README from "../../../../packages/css/src/components/link-list/README.md?raw";
+
+<Meta of={LinkListStories} />
+
+<Markdown>{README}</Markdown>
+
+## Examples
+
+### Default
+
+<Primary />
+
```

Comment:
The Controls are missing here

Reply:
I’ve added them a bit lower now, to the first story for Link.